docs/basics.rst

```diff
@@ -607,6 +607,24 @@ __ http://pypi.python.org/pypi/htmllaundry/
                                      preparer=htmllaundry.sanitize,
                                      validator=colander.Length(1))
 
+If you want to create a subclass of `SchemaNode` and define `preparer` as
+its class variable, you'll need to wrap your preparer in `staticmethod`:
```

Comment:
In reStructuredText, double backticks are for inline code syntax highlighting. Single backticks are for italic or emphasis, unless it is for hyperlinks.

Each of these items needs to have double backticks, e.g., ``SchemaNode``.

While you're at it, would you please also fix the single backticks to double for `preparer` below?